src/app/routes/issues._index.tsx

```diff
@@ -95,7 +95,7 @@ export default function Issues() {
         <div>
             <NavBar login={session.login} role={session.role} />
             <div className='flex flex-col items-center mt-4 mx-2 md:mx-4 '>
-                <Tabs defaultValue={filter} className='w-11/12' onValueChange={(value) => setFilter(value)}>
+                <Tabs defaultValue={filter} className='w-3/5' onValueChange={(value) => setFilter(value)}>
```

Comment:
In order to prevent merge conflicts with #151. This PR already implements that.

```suggestion
                <Tabs defaultValue={filter} className='w-11/12' onValueChange={(value) => setFilter(value)}>
```